tests/test_hwm_pubsub.cpp

```diff
@@ -201,14 +216,32 @@ void test_reset_hwm ()
 
 void test_defaults_1000 ()
 {
-    // send 1000 msg on hwm 1000, receive 1000
-    TEST_ASSERT_EQUAL_INT (1000, test_defaults (1000, 1000));
+    // send 1000 msg on hwm 1000, receive 1000, on TCP transport
+    // repeat the test for both TCP, INPROC and IPC transports:
+
+    TEST_ASSERT_EQUAL_INT (1000, test_defaults (1000, 1000, "tcp://127.0.0.1:*"));
+    TEST_ASSERT_EQUAL_INT (1000, test_defaults (1000, 1000, "inproc://a"));
+    TEST_ASSERT_EQUAL_INT (1000, test_defaults (1000, 1000, "ipc://@tmp-tester"));
```

Comment:
IPC does not work on Windows so it will fail, this line and the other one need ifndef ZMQ_HAVE_WINDOWS